modules/developer-cli-odo-installing-odo-on-macos.adoc

```diff
@@ -6,38 +6,34 @@
 
 = Installing {odo-title} on macOS
 
-== Binary installation
+The `{odo-title}` CLI for macOS is available to download as a binary and as a tarball.
```

Comment:
See all my comments on `Installing odo on Linux` and apply them here to `Installing odo on MacOS`

Reply:
Agreed - same updates made across all OSes